src/common/resourceLoaders/apiTreeLoader.test.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect, default as chai } from "chai";
+import chaiAsPromised from "chai-as-promised";
+
+import { createApiTree } from "./apiTreeLoader";
+import path from "path";
+import tmp from "tmp";
+import fs from "fs-extra";
+import { Name } from "../structures/name";
+
+const validRamlDir = path.join(
+  __dirname,
+  "../../../testResources/raml/mercury"
+);
+
+async function copyMercuryApi(destination: string): Promise<void> {
+  await fs.ensureDir(destination);
+  const exchange = await fs.readFile(path.join(validRamlDir, "exchange.json"));
+  const raml = await fs.readFile(path.join(validRamlDir, "mercury.raml"));
+  await fs.writeFile(path.join(destination, "exchange.json"), exchange);
+  await fs.writeFile(path.join(destination, "mercury.raml"), raml);
```

Comment:
```suggestion
  await fs.copy(validRamlDir, destination);
```

Reply:
I tried this first but it didn't work.  Said it couldn't copy a directory, I will change it to copy the files individually in the followup though